### PR TITLE
Add quantization to Stable Video Diffusion notebook

### PR DESCRIPTION
**Pull request by l-bat**

### CODE REVIEW

File: notebooks/stable-video-diffusion/stable-video-diffusion.ipynb [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "# Image to Video Generation with Stable Video Diffusion\n",
```
Comment (eaidova) [inline notebooks/stable-video-diffusion/stable-video-diffusion.ipynb:9]: <p>could you please check title level for added section. Jugging by Table of content, it looks like you need to Remove one # from Quantization title (for logical separation from Run Video Generation section)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2041/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (eaidova) [inline notebooks/stable-video-diffusion/stable-video-diffusion.ipynb:9]: <pre class="ql-syntax" spellcheck="false">Line #2.    %pip install -q datasets nncf
</pre><p>please specify lower bound for nncf</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2041/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (eaidova) [inline notebooks/stable-video-diffusion/stable-video-diffusion.ipynb:9]: <p>how long data collaction and quantization takes?</p><p>Could you please reduce subset for internal validation adding replacement into notebook  metadata <a href="https://github.com/openvinotoolkit/openvino_notebooks/wiki/Notebooks-Development---CI-Test-Speedup" target="_blank">https://github.com/openvinotoolkit/openvino_notebooks/wiki/Notebooks-Development---CI-Test-Speedup</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2041/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Comment (l-bat) [reply notebooks/stable-video-diffusion/stable-video-diffusion.ipynb:9]: Reduced `subset_size` to 4 samples

Comment (eaidova) [inline notebooks/stable-video-diffusion/stable-video-diffusion.ipynb:9]: <pre class="ql-syntax" spellcheck="false">Line #4.        value=True if quantized_model_present else False,
</pre><p>value = quantized_model_present</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2041/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

**Review by eaidova — APPROVED** [review]

**Review by MaximProshin — APPROVED** [review]
LGTM

File: notebooks/stable-video-diffusion/stable-video-diffusion.ipynb [hunk 2]
```diff
@@ -22,6 +22,12 @@
     "- [Prepare Inference Pipeline](#Prepare-Inference-Pipeline)\n",
     "- [Run Video Generation](#Run-Video-Generation)\n",
     "    - [Select Inference Device](#Select-Inference-Device)\n",
+    " - [Quantization](#Quantization)\n",
```
Comment (nikita-savelyevv) [inline notebooks/stable-video-diffusion/stable-video-diffusion.ipynb:25]:
```suggestion
    "- [Quantization](#Quantization)\n",
```

File: notebooks/stable-video-diffusion/stable-video-diffusion.ipynb (hunk 1 above)
Comment (nikita-savelyevv) [inline notebooks/stable-video-diffusion/stable-video-diffusion.ipynb:9]: <ol><li>How about "That's why we use weight compression" -&gt; "That's why we use <em>only</em> weight compression" ? </li><li><code>vae encoder and decoder</code> -&gt; <code>vae_encoder</code> and <code>vae_decoder</code></li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2041/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Comment (nikita-savelyevv) [inline notebooks/stable-video-diffusion/stable-video-diffusion.ipynb:9]: <ol><li><code>vae encoder and decoder</code> -&gt; <code>vae_encoder</code> and <code>vae_decoder</code></li><li>"The weight compression will be applied to footprint reduction." -&gt; "<em>Only</em> weight compression will be applied <em>for</em> footprint reduction."</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2041/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Comment (nikita-savelyevv) [inline notebooks/stable-video-diffusion/stable-video-diffusion.ipynb:9]: <p>Based on the logs I noticed that only 3 layers are actually compressed. But later I see that the compression ratio is 2x. Do you know, is 2x only coming from compressing embedding layers?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2041/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Comment (l-bat) [reply notebooks/stable-video-diffusion/stable-video-diffusion.ipynb:9]: <p>Because most layers are compressed to 8 bits.</p>

**Review by nikita-savelyevv — APPROVED** [review]

**Pull request merged**